src/test/java/ru/ayaz/UserSocketHandlerTest.java

```diff
@@ -0,0 +1,30 @@
+package ru.ayaz;
+
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.net.Socket;
+
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class UserSocketHandlerTest {
+
+    @Test
+    public void shouldContainMessageInQueueWhenReadFromStreamAndEnqueued() throws IOException, InvalidNicknameException {
```

Comment:
&#x1F34E; **Issue**
You should just throw Exception out of your test method instead of a myriad of them.
This concerns every other test method in your project as well.